apps/condo/domains/banking/schema/BankTransaction.js

```diff
@@ -75,36 +73,37 @@ const BankTransaction = new GQLListSchema('BankTransaction', {
 
         isOutcome: {
             schemaDoc: 'Indicator of outcome transaction which commits negative change to balance',
-            type: Checkbox,
+            type: 'Checkbox',
             isRequired: true,
         },
 
         currencyCode: CURRENCY_CODE_FIELD,
 
         purpose: {
             schemaDoc: 'Textual description of payment purpose in free form',
-            type: Text,
+            type: 'Text',
             isRequired: true,
         },
 
         meta: {
             schemaDoc: 'Stores data, obtained from external source',
-            type: Json,
+            type: 'Json',
             isRequired: true,
         },
 
         importId: {
             schemaDoc: 'Identifier of corresponding record in external system, from that this record was copied',
-            type: Text,
+            type: 'Text',
             isRequired: true,
         },
 
         importRemoteSystem: {
             schemaDoc: 'Name of external system, from that this record was copied',
-            type: Select,
+            type: 'Select',
             options: IMPORT_REMOTE_SYSTEM_VALUES,
             // Cannot use `enum` datatype because a number is presented in first character of a value "1CClientBankExchange".
             dataType: 'string',
+            graphQLReturnType: 'String',
```

Comment:
?

Reply:
`enum` cannot be generated because one of the values ​​starts with a digit (`1CClientBankExchange`).
Such values ​​are not allowed for `enum`